### DISCUSSION TITLE
Add a feature for managing your account like creating and managing your personal token.

### DISCUSSION THREAD

**Discussion by Zyad205** [community/community#59364] [Mobile] [answered]
Add a feature for managing your account like creating and managing your personal token.

**Comment by stevepopovich** [comment]
Hi @Zyad205 what are you looking for here specifically? Is there a use-case you had in mind?

**Reply by Zyad205** [reply]
It would be nice to access a trusted platform to make a personal access token if I am going to use a laptop that is not mine, or manage my repo right now there is not a lot of functionality.

**Reply by stevepopovich** [reply, marked as answer]
Gotcha, ok thanks, I passed this onto the team for prioritization.
